code/modules/surgery/healing.dm

```diff
@@ -49,107 +52,164 @@
+		"<span class='notice'>[user] fixes some of [target]'s wounds.</span>", TRUE)
+	var/urdamageamt_burn = brutehealing * 0.8
+	var/urdamageamt_brute = burnhealing * 0.8
+	if(missinghpbonus)
+		urdamageamt_brute += round((target.getBruteLoss()/ (missinghpbonus*2)),0.1)
+		urdamageamt_burn += round((target.getFireLoss()/ (missinghpbonus*2)),0.1)
+
+	target.take_bodypart_damage(urdamageamt_brute, urdamageamt_burn)
 	return FALSE
 
 /***************************BRUTE***************************/
 /datum/surgery/healing/brute
 	name = "Tend Wounds (Bruises)"
 
 /datum/surgery/healing/brute/basic
+	name = "Tend Wounds (Bruises, Basic)"
 	replaced_by = /datum/surgery/healing/brute/upgraded
 	healing_step_type = /datum/surgery_step/heal/brute/basic
+	desc = "A surgical procedure that provides basic treatment for a patient's brute traumas. Heals slightly more when the patient is severely injured."
 
 /datum/surgery/healing/brute/upgraded
+	name = "Tend Wounds (Bruises, Adv.)"
 	replaced_by = /datum/surgery/healing/brute/upgraded/femto
 	requires_tech = TRUE
 	healing_step_type = /datum/surgery_step/heal/brute/upgraded
+	desc = "A surgical procedure that provides advanced treatment for a patient's brute traumas. Heals more when the patient is severely injured."
 
 /datum/surgery/healing/brute/upgraded/femto
+	name = "Tend Wounds (Bruises, Exp.)"
 	replaced_by = /datum/surgery/healing/combo/upgraded/femto
 	requires_tech = TRUE
 	healing_step_type = /datum/surgery_step/heal/brute/upgraded/femto
+	desc = "A surgical procedure that provides experimental treatment for a patient's brute traumas. Heals considerably more when the patient is severely injured."
 
 /********************BRUTE STEPS********************/
 /datum/surgery_step/heal/brute/basic
 	name = "tend bruises"
 	brutehealing = 5
+	missinghpbonus = 15
 
 /datum/surgery_step/heal/brute/upgraded
-	brutehealing = 10
+	brutehealing = 5
+	missinghpbonus = 10
 
 /datum/surgery_step/heal/brute/upgraded/femto
-	brutehealing = 15
+	brutehealing = 5
+	missinghpbonus = 5
 
 /***************************BURN***************************/
 /datum/surgery/healing/burn
 	name = "Tend Wounds (Burn)"
 
 /datum/surgery/healing/burn/basic
+	name = "Tend Wounds (Burn, Basic)"
 	replaced_by = /datum/surgery/healing/burn/upgraded
 	healing_step_type = /datum/surgery_step/heal/burn/basic
+	desc = "A surgical procedure that provides basic treatment for a patient's burns. Heals slightly more when the patient is severely injured."
 
 /datum/surgery/healing/burn/upgraded
+	name = "Tend Wounds (Burn, Adv.)"
 	replaced_by = /datum/surgery/healing/burn/upgraded/femto
 	requires_tech = TRUE
 	healing_step_type = /datum/surgery_step/heal/burn/upgraded
+	desc = "A surgical procedure that provides advanced treatment for a patient's burns. Heals more when the patient is severely injured."
 
 /datum/surgery/healing/burn/upgraded/femto
+	name = "Tend Wounds (Burn, Exp.)"
 	replaced_by = /datum/surgery/healing/combo/upgraded/femto
```

Comment:
```suggestion
	replaced_by = null
```

Reply:
pardon

Reply:
@alexkar598 I don't get why?

Reply:
```/datum/surgery/healing/combo/upgraded/femto //no real reason to type it like this except consistency, don't worry you're not missing anything
	name = "Tend Wounds (Mixture, Exp.)"
	replaced_by = null```

Its used like this elsewhere,it doesnt seem like a bad idea to not make a thing be replaced by itself. Sounds runtime/issue prone

Reply:
it's not itself though, `/datum/surgery/healing/burn/upgraded/femto` gets replaced by `/datum/surgery/healing/combo/upgraded/femto`

Reply:
this is stupid and you misread